tests/rptest/tests/scaling_up_test.py

```diff
@@ -381,4 +383,112 @@ def initial_rebalance_finished():
+                          redpanda_remote_read=True)
+
+        total_replicas = 3 * partition_cnt
+
+        self.client().create_topic(topic)
+        self.logger.info(
+            f"Producing {data_size} bytes of data in {msg_cnt} total messages")
+        self.producer = KgoVerifierProducer(
+            self.test_context,
+            self.redpanda,
+            topic.name,
+            msg_size=msg_size,
+            msg_count=msg_cnt,
+            custom_node=self.preallocated_nodes)
+        self.producer.start()
+        self.producer.wait()
+
+        # add fifth node
+        self.redpanda.start_node(self.redpanda.nodes[4])
+        self.wait_for_partitions_rebalanced(total_replicas=total_replicas,
+                                            timeout_sec=self.rebalance_timeout)
+
+        def print_disk_usage(usage):
+            for n, b in usage.items():
+                self.logger.info(
+                    f"node: {n} total partitions size: {b/(1024*1024)} Mb")
+
+        def verify_disk_usage(usage: dict, added_ids: list, percentage: float):
+            old_nodes_usage = [
+                b for id, b in usage.items() if id not in added_ids
+            ]
+            avg_usage = sum(old_nodes_usage) / len(old_nodes_usage)
+
+            for id in added_ids:
+                added_node_usage = usage[id]
+                assert added_node_usage < percentage * avg_usage, \
+                f"Added node {id} disk usage {added_node_usage} is to large,"
+                "expected usage to be smaller than {percentage * avg_usage} bytes"
+
+        usage = self._kafka_usage(nodes=self.redpanda.nodes[0:5])
+        print_disk_usage(usage)
+
+        verify_disk_usage(usage,
+                          [self.redpanda.node_id(self.redpanda.nodes[4])], 0.2)
+
+        # add sixth node
+        self.redpanda.start_node(self.redpanda.nodes[5])
+        self.wait_for_partitions_rebalanced(total_replicas=total_replicas,
+                                            timeout_sec=self.rebalance_timeout)
+
+        usage = self._kafka_usage(self.redpanda.nodes)
+        print_disk_usage(usage)
+        verify_disk_usage(usage, [
+            self.redpanda.node_id(self.redpanda.nodes[4]),
+            self.redpanda.node_id(self.redpanda.nodes[5])
+        ], 0.2)
+        # verify that data can be read
+        self.consumer = KgoVerifierSeqConsumer(self.test_context,
+                                               self.redpanda,
+                                               topic.name,
+                                               msg_size,
+                                               nodes=self.preallocated_nodes)
+
+        self.consumer.start(clean=False)
+        self.consumer.wait()
+
+        assert self.consumer.consumer_status.validator.invalid_reads == 0, \
+        f"Invalid reads in topic: {topic.name}, invalid reads count: "
+        "{self.consumer.consumer_status.validator.invalid_reads}"
```

Comment:
hmm, not sure this will work, this should be an `f""` string as well, right?

Reply:
also I guess without a `\` on the previous line this is just a statement on its own?

Reply:
@mmaslankaprv did you see these comments?